sirepo/raydata/scan_monitor.py

```diff
@@ -356,18 +356,16 @@ async def _process_analysis_queue():
             s = _AnalysisStatus.ERROR
             try:
                 _Analysis.set_scan_status(d, _AnalysisStatus.RUNNING)
-                with pkio.save_chdir(d.get_output_dir(), mkdir=True), pkio.open_text(
-                    "run.log", mode="w"
-                ) as l:
+                with pkio.save_chdir(d.get_output_dir(), mkdir=True):
                     try:
                         for n in d.get_notebooks():
                             p = await asyncio.create_subprocess_exec(
-                                "papermill",
+                                cfg.papermill_script,
                                 str(n.input_f),
                                 str(n.output_f),
-                                *d.get_papermill_args(),
-                                stderr=asyncio.subprocess.STDOUT,
-                                stdout=l,
+                                str(asyncio.subprocess.STDOUT),
```

Comment:
I think we want to continue to use the stdin and stdout kwargs that create_subprocess_exec has. That way all output from the subrocess is sent to it. In the way the code is written here only papermill logs will go to the log file. But, things like setting up the conda env (which we'll add) will go to the scan monitor logs. I think that could make debugging confusing and the scan_monitor doesn't need to know about these logs. Something like,
```python
with pkio.save_chdir(d.get_output_dir(), mkdir=True), pkio.open_text(
    "run.log", mode="w"
) as l:
    try:
        for n in d.get_notebooks():
            s = 'execute-analysis.sh' # TODO: def as constant above
            pkjinja.render_resource(_PAPERMILL_SCRIPT, PKDict(input_f=str(n.input_f), ...) output=d.join(s)
            p = await asyncio.create_subprocess_exec(
                s,
                stderr=asyncio.subprocess.STDOUT,
                stdout=l,
            )
```

Reply:
7c7866ddefbd0a7acf2fcf3cbe7a2d8845de81b3